Training/2.0 Libraries.md

```diff
@@ -25,14 +26,19 @@ Make a two-dimensional numpy array. First row will be: [1,2,3] and second row wi
 Using numpy functionality find the mean, minimum and max value of your array. Index the maximum value and assign it to a variable named max. Index the minimum value and assign it to a variable named min. 
 
 ```
+arr = np.array([[1,2,3],[4,5,6]])
+max  = np.argmax(arr)
+min = np.argmin(arr)
 
 ```
 
 ### Task 2b)
 Calculate the trimmed mean of the following array using ONLY Numpy and python's base functions: [67,67,65,50,69,68,67,70,69,66]. The trimmed mean is the mean of the data with the largest and the smallest values (outliers) removed.
 
 ```
-
+arr = arr.remove(np.max(arr))
+arr = arr.remove(np.min(arr))
+trimmed_mean = np.mean(arr)
```

Comment:
Good attempt, but .remove() method only works on lists, not numpy arrays. 
The correct solution would be to use numpy.delete:
`arr = numpy.delete(arr,np.argmax(arr))`
`arr = numpy.delete(arr,np.argmin(arr))`